staging_manifest.yml

```diff
@@ -2,17 +2,18 @@
 applications:
 - name: federalist-staging
 buildpack: https://github.com/cloudfoundry/nodejs-buildpack#v1.5.4
+stack: cflinuxfs2
 domain: fr.cloud.gov
 disk_quota: 2G
-memory: 512MB
+memory: 1G
 instances: 2
 services:
 - federalist-staging-rds
 - federalist-staging-redis
 - federalist-staging-s3
 - federalist-staging-env
 env:
-  NODE_ENV: staging
+  NODE_ENV: production
   SAILS_LOG_LEVEL: verbose
-  NPM_CONFIG_PRODUCTION: false
+  NPM_CONFIG_PRODUCTION: true
```

Comment:
When deploying on :cloud:.gov, you need to ensure that your running `NODE_ENV: production` and `NPM_CONFIG_PRODUCTION: true` otherwise you're going to be installing some `devDependencies`. The installation of certain libraries will hit the memory and disk limits of the container that it is being deployed to.